cli/src/validate.rs

```diff
@@ -270,3 +281,20 @@ fn print_validation_level(level: JunitValidationLevel) -> ColoredString {
         JunitValidationLevel::Valid => "VALID".green(),
     }
 }
+
+fn print_codeowners(codeowners: Option<CodeOwners>) {
+    println!("\nChecking for codeowners file...");
+    match codeowners {
+        Some(owners) => {
+            println!(
+                "  {} - Found codeowners:",
+                print_validation_level(JunitValidationLevel::Valid)
+            );
+            println!("    Path: {:?}", owners.path);
```

Comment:
` "/home/hui/dev/analytics-cli/./CODEOWNERS"`

Can we normalize this path? You can probably call `.canonicalize()` in `find_file`

Reply:
fixed!